doc/source/docsportal_sec.rst.inc

```diff
@@ -0,0 +1,199 @@
+care about avoiding placing sensitive information from git repositories is
+taken and it is explicitly in the responsibility of each individual project
+maintainers to ensure no sensitive information land in git.
+
+Ensuring, that only checked and approved content will be merged in git and
+published is a primary responsibility of :ref:`Zuul` which applies also
+:ref:`git_control` to manage required conditions that need to be fulfilled on
+every pull request of every project.
+
+Separation
+----------
+
+The project is implemented as a combination of multiple software
+and solution components communicating with each other. Those
+components are installed physically separated from each other
+with no direct connectivity except of public HTTPS.
+
+* Web Server
+
+  * Nginx web server accepts HTTP protocol requests. It rewrites
+    the request to a destination on the Remote Storage.
+
+  * It performs remote request to fetch the requested content
+    and serves it back to the initial requestor.
+
+  * Depending on the content accessibility web server is either exposed
+    directly to the web or behind additional reverse proxy implementing 
+    required security limitations.
+
+* Storage
+
+  * OpenStack Swift storage. Practically this can be any other object storage
+    which allows web access.
+
+  * Zuul CI/CD writes approved content into the storage
+    destination.
+
+  * Web Server fetches the content.
+
+  * The content in the Storage for the Documentation Portal is by
+    definition public content (not protected by any additional ACLs).
+
+  * If the content is not designed to be publicly available content in the
+    Storage must be protected by ACLs. This in turn will require enabling web
+    server to access this content (i.e. swift-proxy in the case of using
+    OpenStack Swift).
+
+* :ref:`zuul`
+
+  * Zuul installation manages git projects and implements
+    configured CI rules in order to ensure that only checked and
+    approved content will be merged. Default configuration
+    forbids anybody (except of Zuul Administrators) to bypass
+    required checks and merge content manually
+  * Once all the prerequisites are fulfilled Zuul merges Pull
+    Request, builds documentation and pushes it to storage with
+    dedicated credentials.
+  * Only git projects explicitly included in the Zuul tenant are
+    being respected. Registered git projects with disabled
+    branch protection rules are ignored.
+
+* `GitHub <https://github.com>`_
+
+  * An external git hosting provider.
+  * Projects in the GitHub organization are managed by `dedicate process
+    <https://docs.otc-service.com/system-config/gitcontrol.html>`_
+
+Interface Description
+---------------------
+
+The only public facing interface is the regular Web using HTTPS (automatic
+forwarding from HTTP).
+
+Tenant Security
+---------------
+
+Documentation Portal is not providing any tenant support. All documents that are hosted
```

Comment:
What you mean by Documentation portal is not providing any tenant support?
Documentation portal is basically support for customers just by its definition.
Maybe only some rephrasing is needed or better explanation